### PR TITLE
Review

### PR DESCRIPTION
**Pull request by KingdomOfEnds**
@PeterTurcan 

Looks great overall! Few tweaks:

1. Removed the private testing credentials (obviously you would have) but did anyway.
1. Added a `package.json` with dependencies.
1. Made the use of ES6 `const` and `let` variables more consistent. Removed `var`. The use of `var` versus `let` here is a little subjective. I'd encourage consistent use of `const` since it protects the number values from being modified at write and run time.
1. Lastly, moved all function syntax to be the same.

So, the only major thing I did here was add in a `package.json` file :)

Confirmed working 100% - I know there are a few other items to be added and tested too :)

image

### CODE REVIEW

File: device/app.js [hunk 1]
```diff
@@ -6,31 +6,30 @@ const chalk = require('chalk');
 console.log(chalk.yellow('WineCellar device app'));
 
 // The device connection string to authenticate the device with your IoT hub.
-var connectionString = 'HostName=PetersIOTHub.azure-devices.net;DeviceId=PetersDevice;SharedAccessKey=O9MRAgElFD2bLU9t4+5fcPGR/zqtAinwOtKdPhVTXQg=';
```
Comment (KingdomOfEnds) [inline device/app.js:9]: Removed private testing credentials.

File: device/app.js [hunk 2]
```diff
@@ -6,31 +6,30 @@ const chalk = require('chalk');
...
 // The device connection string to authenticate the device with your IoT hub.
-var connectionString = 'HostName=PetersIOTHub.azure-devices.net;DeviceId=PetersDevice;SharedAccessKey=O9MRAgElFD2bLU9t4+5fcPGR/zqtAinwOtKdPhVTXQg=';
-// var connectionString = '<your device connection string>';
+const connectionString = '<your device connection string>';
 
 // The sample connects to a device-specific MQTT endpoint on your IoT Hub.
-var Mqtt = require('azure-iot-device-mqtt').Mqtt;
-var DeviceClient = require('azure-iot-device').Client
-var Message = require('azure-iot-device').Message;
+const Mqtt = require('azure-iot-device-mqtt').Mqtt;
```
Comment (KingdomOfEnds) [inline device/app.js:12]: Consistent use of `const` and `let` throughout (ES6 variable convention).

File: hub/app.js [hunk 3]
```diff
@@ -7,23 +7,18 @@ const chalk = require('chalk');
 console.log(chalk.yellow('WineCellar Operator: the back-end service app'));
 
 // The connection string for the IoT Hub service.
-var connectionString = 'HostName=PetersIOTHub.azure-devices.net;SharedAccessKeyName=iothubowner;SharedAccessKey=1fRjAms3Y/U4YUs2HvTLSUyeKxGd0hpSe1Cat8792UQ=';
```
Comment (KingdomOfEnds) [inline hub/app.js:10]: Removed private testing credentials.

File: hub/app.js [hunk 4]
```diff
@@ -7,23 +7,18 @@ const chalk = require('chalk');
 console.log(chalk.yellow('WineCellar Operator: the back-end service app'));
 
 // The connection string for the IoT Hub service.
-var connectionString = 'HostName=PetersIOTHub.azure-devices.net;SharedAccessKeyName=iothubowner;SharedAccessKey=1fRjAms3Y/U4YUs2HvTLSUyeKxGd0hpSe1Cat8792UQ=';
-var deviceId = 'PetersDevice';
-//var connectionString = '<your iothubowner connection string>';
-//var deviceId = '<your device ID>';
-
+const connectionString = '<your iothubowner connection string>';
```
Comment (KingdomOfEnds) [inline hub/app.js:10]: Consistent use of `const` and `let` (ES6 variable syntax).

File: hub/app.js [hunk 5]
```diff
@@ -116,8 +111,8 @@ registry.getTwin(deviceId, function (err, twin) {
     }
 });
 
-var queryTwins = function () {
-    var query = registry.createQuery("SELECT * FROM devices WHERE tags.cellar = 'Cellar1'", 100);
+function queryTwins() {
```
Comment (KingdomOfEnds) [inline hub/app.js:114]: Consistent use of function syntax.

File: package.json [hunk 6]
```diff
@@ -0,0 +1,15 @@
+{
```
Comment (KingdomOfEnds) [inline package.json:1]: Added `package.json` with file dependencies.

**Review by PeterTurcan — APPROVED** [review]

**Pull request merged**